src/communication/xcWebSocketSubscriber.ts

```diff
@@ -230,14 +240,14 @@ function isSubscribed(subscribedStateMachines, componentName, stateMachineName)
 }
 
 
-function isSameComponent(jsonData, codes) {
-    let sameComponent = jsonData.stateMachineRef.ComponentCode === parseInt(codes.componentCode);
+function isSameComponent(jsonData, componentCode) {
```

Comment:
for the next migration step: this function looks hard to understand... of it is only used once, and it just navigates on the json, I wonder if it isn't simpler just to inline it.